website/source/docs/agent/configuration/tls.html.md

```diff
@@ -67,6 +67,9 @@ the [Agent's Gossip and RPC Encryption](/docs/agent/encryption.html).
 - `tls_min_version` - Specifies the minimum supported version of TLS. Accepted
   values are "tls10", "tls11", "tls12". Defaults to TLS 1.2.
 
+- tls_prefer_server_cipher_suites - This option will cause Nomad to prefer the
```

Comment:
Put it in code blocks and always start the docs with `Specifies...`. Here it would be `Specifies whether TLS connections should prefer the server's ciphersuite over the client's.`

Reply:
Add the default value to this, tls_min_version and tls_cipher_suites. For the cipher_suites use: https://www.nomadproject.io/docs/agent/configuration/client.html#servers

Reply:
tls_cipher_suites has the default set- it currently is maintained in the TLS config file so that TLS logic stays in one file.

Reply:
tls_min_version and tls_cipher_suites both specify defaults, but if there is a clearer way to indicate this let me know.

Reply:
See how all other fields use the common format of specifying defaults after the parameter name: https://github.com/hashicorp/nomad/pull/4338/files#diff-3b31a52d9f28cef5fa96fb6f8b149907R57